docs/plugin-proposal-pipeline-operator.md

```diff
@@ -22,31 +22,60 @@ $ npm install --save-dev @babel/plugin-proposal-pipeline-operator
 }
 ```
 
-The Pipeline Proposal is one of three competing implementations. Which implementation the plugin should use is configured with the `"proposal"` option. This option is required and should be one of:
+The pipeline proposal is one of three competing implementations. Configure with proposal to use with the required `"proposal"` option.
```

Comment:
```suggestion
The pipeline proposal is one of four competing implementations. Configure with proposal to use with the required `"proposal"` option.
```

4 now?

Reply:
I guess four is correct. Since smart-mix was deprecated, I wasn't thinking of it as a proposal, but it still exists, I suppose.